packages/frontend/app/server.ts

```diff
@@ -21,6 +22,7 @@ export default (options: any) => {
 
 // this is the actual production server
 if (process.env.NODE_ENV === 'production') {
+    logger.info('dotcom-rendering is GO.');
```

Comment:
It's good for testing, but I'd say probably we should only log errors/issues in PROD. But I have quite strong opinions on this(!) - I've struggled under a lot of noisy logging in the past.

Reply:
I think this is useful so that people can see an example of dotcom-rendering logs in ELK - otherwise it's totally silent and people could assume that it doesn't have any logging capability. It could be removed once the app becomes more chatty - but it only  gets logged once on each deploy.